static/build/.tmp_update/runtime.sh

```diff
@@ -344,6 +357,9 @@ launch_game() {
     # Free memory
     $sysdir/bin/freemma
 
+    # Reset networking
+    touch /tmp/network_changed
```

Comment:
This should be behind a condition I think - and actually test if any network services are enabled - and whether `keepAlive` is disabled

Reply:
all these checks happen in update_networking check() (which is what's eventually called by touching this flag),  no?

Reply:
The `network_changed` flag was added to avoid doing the checks all the time - this negates that